pydoc/getting_started/introduction/features.md

```diff
@@ -0,0 +1,29 @@
+# Features
```

Comment:
There is already a doc/features.md file.

Reply:
This was copied over to have the documentation build nicer and check that the ToC works as intended. It can be removed or made into a "include" document instead?